crates/zserio-rs-build/README.md

```diff
@@ -0,0 +1,35 @@
+# zserio-cli
```

Comment:
```suggestion
# zserio-rs-build
```
I think the naming is currently a inconsistent. There are some places where the crate is called `zserio-rs-build`, and in other cases `zserio-cli`.

Reply:
Good catch, there were two places that sitll used the old name. I've fixed both of them.